packages/api/src/ai/facades/ai.service.ts

```diff
@@ -94,18 +95,24 @@ Helpful answer:`
     }
   }
 
-  async askAiToDescribeDocument(lcDocuments: Document[]): Promise<{
+  async askAiToDescribeDocument(
+    lcDocuments: Document[],
+    chatLlm: string
+  ): Promise<{
     name: string;
     description: string;
   }> {
     try {
       const titleChain = new LLMChain({
-        llm: this.llmModel,
+        llm: await this.chatModelService.getChatModel(chatLlm),
```

Comment:
In this function, extract the `await this.chatModelService.getChatModel(chatLlm)` to a const